src/services/api.ts

```diff
@@ -77,9 +87,22 @@ export const getApi = (token?: string) => {
     return { state, data: content, status: response.status };
   };
 
+  let userReactors: Channel[] | undefined = undefined;
+  chrome.storage.local.get('userReactors', function (result) {
```

Comment:
I was thinking we move this to the util or some kind of service?

Reply:
I don't think the userReactors variable will update after initialization. Probably works fine since getReactionVideos is ran in a separate app and gets refreshed, but this function has unintended side effects if used anywhere else.

We could just run the user reactors from chrome storage each time getReactionVideos is called (don't think that would add much overhead since it's just pulling from localStorage). Might even be worth explicitly passing userReactors in. eg. `getReactionVideos(id: string, channels?: Channels[])` that way it's clear from the caller's perspective of what's happening and they're able to get all videos if needed.